tests/test_main.py

```diff
@@ -363,6 +364,13 @@ def test_160_check_only_disabled_one_check(self):
             expected_errors.pop(disable_error)
             self.assertDictEqual(real_errors, expected_errors)
 
+    def test_165_no_raises_unlink(self):
+        extra_params = ["--disable=all", "--enable=no-raise-unlink"]
+        self.assertDictEqual(
+            self.run_pylint([os.path.join(self.root_path_modules, "eleven_module")], extra_params).linter.stats.by_msg,
+            {"no-raise-unlink": 2},
```

Comment:
Is this for v11?

Take into account this lint only applies for version 15.0 and above.

Reply:
I moved it out to test_module which still has a 10.0 manifest but it seems like that is not very well respected, still may seem less confusing since it has no version in its name.

Also added more code to the test case to ensure this check only runs on 15.0 and above.